.github/workflows/unittest.yml

```diff
@@ -0,0 +1,114 @@
+name: UnitTests
+
+on: [push, pull_request]
+
+jobs:
+  ubuntu:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+        use-conda: [true, false]
+        context: ['fork', 'spawn', 'forkserver']
+        use-dist: [false]
+        scikit-learn: [0.22.2, 0.23.1]
+        include:
+          - python-version: 3.8
+            code-cov: true
+            scikit-learn: 0.22.2
+            context: 'fork'
+            use-dist: false
+          - python-version: 3.7
+            scikit-learn: 0.22.2
+            use-conda: false
+            use-dist: true
+            context: 'fork'
+          - python-version: 3.6
+            use-conda: false
+            scikit-learn: 0.19.2
+            context: 'fork'
+            use-dist: false
+        exclude:
+          # Older python cannot move threadlock objects around
+          - python-version: 3.6
```

Comment:
Are these single tests that fail and could be skipped individually?

Reply:
I have enabled all test on 3.6 using a pickable mock strategy.